### DISCUSSION TITLE
How do you clean uninstall?

### DISCUSSION THREAD

**Discussion by PlankyTheNoob** [filips123/PWAsForFirefox#158] [Q&A] [answered]
I downloaded it just to test it out, but how do you clean uninstall it? I was asked to download some apps, and after I uninstall it, there is still a folder in "Roaming" called FirefoxPWA, and I was wondering if it's safe to uninstall.

**Comment by filips123** [comment, marked as answer]
Before you uninstall the program, you should first uninstall all web apps you created. Then, you can uninstall the program and remove that folder.

If you have already uninstalled the program, it should also be fine if you just delete web app shortcuts (in `%AppData%\Microsoft\Windows\Start Menu\Programs` on Windows) instead of reinstalling the program just to uninstall the apps.
